docassemble/ALToolbox/data/questions/income.yml

```diff
@@ -0,0 +1,10 @@
+---
+variable name: al_times_per_year_list
+data:
+  - [12, "Monthly"]
```

Comment:
This is a short list so I recommend sorting by frequency. Eg., make "weekly" first and then ascending size order, with "yearly" last.